compiler/luci-interpreter/src/loader/GraphLoader.cpp

```diff
@@ -199,16 +209,54 @@ void GraphLoader::loadOperators()
   KernelBuilder kernel_builder(_graph_to_runtime_graph, _node_to_tensor);
 
   // Create kernels for executable nodes. This has to be done in execution order.
-  for (const loco::Node *loco_node :
-       loco::postorder_traversal(loco::output_nodes(const_cast<loco::Graph *>(_graph))))
+  auto graph = const_cast<loco::Graph *>(_graph);
+
+  auto const graph_nodes = loco::all_nodes(graph);
+
+  // Checking for execution plan in node annotations.
+  bool has_execution_annotation = true;
+  auto const checking_exec_plan = [&has_execution_annotation](auto const node) {
+    const auto *circle_node = loco::must_cast<const luci::CircleNode *>(node);
+    if (!luci::has_execution_plan(circle_node))
+      has_execution_annotation = false;
+  };
+  std::for_each(begin(graph_nodes), end(graph_nodes), checking_exec_plan);
+
```

Comment:
(optional)

`std::all_of` returns `true` if predicat (third arg) is `true` for all elements of container.  

```suggestion
  bool has_execution_annotation = std::all_of(begin(graph_nodes), end(graph_nodes), luci::has_execution_plan);
```

Reply:
I understand but `has_execution_plan` should have ` luci::CircleNode *` but  `graph_nodes` contains `loco::Node *` and that is why we should use `loco::must_cast`